lib/exceptions/backends/log_result.rb

```diff
@@ -12,8 +12,9 @@ def initialize(backend, logger = ::Logger.new(STDOUT))
         @logger  = logger
       end
 
-      def notify(exception, options = {})
-        backend.notify(exception, options).tap do |result|
+      def notify(*args)
+        backend.notify(*args).tap do |result|
```

Comment:
here's the problem: calling `Exceptions.notify(error_class: "Blah")` will produce a blank error: https://rollbar.com/Remind/r101-api/items/847/

you can see the reason for this in this code example:

```ruby
class Rollbar
  def notify(exception = nil, error_class: nil, error_message: nil,
             parameters: {}, context: {})
    exception || error_class
  end
end

class Wrapper
  def initialize(wrapped)
    @wrapped = wrapped
  end

  def notify(exception, args = {})
    @wrapped.notify(exception, args)
  end
end

w = Wrapper.new(Rollbar.new)

p w.notify(error_class: "MyCustomError")
```
this prints: `{:error_class=>"MyCustomError"}`

that's not what we want. we want it to print `"MyCustomError"`. the issue is that when there aren't any positional arguments specified, the hash argument gets passed in as the first argument instead of the second argument.

this fixes the problem:
```ruby
class Rollbar
  def notify(exception = nil, error_class: nil, error_message: nil,
             parameters: {}, context: {})
    exception || error_class
  end
end

class Wrapper
  def initialize(wrapped)
    @wrapped = wrapped
  end

  def notify(*args)
    @wrapped.notify(*args)
  end
end

w = Wrapper.new(Rollbar.new)
p w.notify(error_class: "MyCustomError")
```
prints out: `"MyCustomError"`